src/main/java/org/agoncal/application/petstore/view/ViewUtils.java

```diff
@@ -52,7 +52,7 @@ public static String display(Object object)
                   // Find a matching getter and invoke it to display the key
                   for (Method method : object.getClass().getDeclaredMethods())
                   {
-                     if (method.equals(new PropertyDescriptor(field.getName(), object.getClass()).getReadMethod()))
```

Comment:
@andxu - we should not change this line. The scenario is all about migrating an existing Java EE app to Azure. If you must change this, then we should contribute to the upstream repo owned by Antonio and then we should absorb it here